.travis/panic-test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Check the unwrap and panic function presenting in src/ directory
+# of the repo. Testing suite is excluded. If such a function presents
+# return exit 1 for fail, otherwise return exit 0 for sucess.
+
+# Getting the banned list from banned functions
+banned_list=()
+banned_function="$1"
+while read -r line; do
+    banned_list=(${banned_list[@]} "$line")
+done < "$banned_function"
+
+output=""
+panic_line=0
```

Comment:
0 isn't a good sentinel.  Consider what'll happen if a banned function appears on the first line of a file.

Reply:
Changed to using boolean as a sentinel instead of using 0.